powersimdata/export/export_to_pypsa.py

```diff
@@ -317,10 +163,7 @@ def export_to_pypsa(
         generators to the exported pypsa network. This ensures feasibility when
         optimizing the exported pypsa network as is. The default is True.
     """
-    from powersimdata.scenario.scenario import Scenario  # avoid circular import
-
-    if not PYPSA_AVAILABLE:
-        raise ImportError("PyPSA is not installed.")
+    pypsa = _check_import("pypsa")
```

Comment:
Since `pypsa` is listed as a dev package, I have removed the import at the beginning of the module and imported and used the `_check_import` utility function. I believe it is cleaner than just importing `pypsa` and getting a `ImportError`